lib/compute/operation.js

```diff
@@ -160,6 +166,7 @@ function Operation(scope, name) {
   this.completeListeners = 0;
   this.hasActiveListeners = false;
   this.name = name;
+  this.progress = 0;
```

Comment:
Maybe we shouldn't initialize this to 0, as one could get a list of operations from `getOperations()` and printing them out would show 0, even if they were completed (while metadata.progress would be 100). We should probably only set progress every time we do a `getMetadata`.